app/models/ProfessionalBody.scala

```diff
@@ -22,6 +22,10 @@ import play.api.i18n.Messages
 case class ProfessionalBody(name: String, synonyms: List[String], startYear: Option[Int]) {
   def toAutoCompleteJson(implicit messages: Messages): JsObject =
     Json.obj("displayName" -> name, "synonyms" -> synonyms)
+
+  def toDisplayText(): String = {
+    startYear.foldLeft(name){(psubName,year) => psubName + ", with effect from 6 April " + year}
+  }
 }
```

Comment:
Could this be refactored to this:

def toDisplayText: String = if (startYear.isDefined) name + ", with effect from 6 April " + startYear.get else name

Also would it be possible to remove the empty parenthesis please